run_scans.py

```diff
@@ -194,7 +220,6 @@ def launchArgs(tool, shelf, slot, link, chamber, vfatmask, scanmin, scanmax, nev
     print list(
             itertools.izip([options.tool for x in range(len(chamber_config))],
                          [options.shelf for x in range(len(chamber_config))],
-                         [options.slot for x in range(len(chamber_config))],
```

Comment:
it seems like from this you have not updated this printout with the new options you've added.

As a non-exhaustive example you've removed slot, but `--voltageStepPulse` is not present.